checkstyle-tester/pom.xml

```diff
@@ -74,6 +74,12 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-jxr-plugin</artifactId>
             <version>${jxr-plugin.version}</version>
+            <!-- plugin do not have ability to define followinng by system property -->
+            <configuration>
+              <excludes>
+                <exclude>**/.ci-temp/**/*</exclude>
```

Comment:
Shouldn't we put local checkstyle outside the path of the rest of contribution? isn't that the issue here?

Reply:
Please rephrase question, I do not understand it.
This fix is unfortunate, I do not know how to workaround it.

Reply:
> [DEBUG] parsing... local-checkstyle/.ci-temp/contribution/patch-diff-report-tool/src/test/resources/run/NonCompilable.java

Errors are from trying to process contribution's test Java files. They aren't real files.

This can only be because checkstyle-local is picking up contribution since contribution is cloned into local checkstyle's workspace.
If we move contribution clone to another folder and not put it in into checkstyle-local, then we won't need this override.

Can we clone contribution to something like `/temp/`?

Reply:
We don't need to make the folder path that specific, but I still think all this should be done inside the repo folder.